extensions/amp-story/1.0/amp-story.js

```diff
@@ -1455,6 +1460,39 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @private
+   */
+  openMask_() {
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
+    if (!this.maskElement_) {
+      const mask = this.win.document.createElement('div');
+      mask.classList.add('i-amphtml-amp-story-sidebar-mask');
+      mask.addEventListener('click', () => {
+      actions.execute(this.sidebar_, 'close', /* args */ null,
+        /* source */ null, /* caller */ null, /* event */ null,
+            ActionTrust.HIGH);
+      });
+      // The mask div should be added right after the sidebar.
+      const referenceNode = this.win.document.getElementsByTagName('amp-sidebar')[0];
+      referenceNode.parentNode.insertBefore(mask, referenceNode.nextSibling);
+      mask.addEventListener('touchmove', e => {
```

Comment:
Why is this needed? If you're trying to prevent other things from happening, like the "tap next" navigation UI help, there are better ways to do it, like adding the `SIDEBAR_STATE` to this list: https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story.js#L611
Lmk if you had to use this for another reason that I missed :)